fe/fe-core/src/main/java/com/starrocks/authentication/UserAuthenticationInfo.java

```diff
@@ -51,6 +53,8 @@ public class UserAuthenticationInfo implements Writable {
     @Expose(serialize = false)
     protected PatternMatcher hostPattern;
 
+    protected Map<String, Object> extraInfo = new HashMap<>();
```

Comment:
What does extraInfo stand for? please add a comment

Reply:
done